src/testing/redux-factory.ts

```diff
@@ -0,0 +1,96 @@
+import { Factory } from "fishery";
```

Comment:
This is the first pass setting up the mock for the model watcher data for the model info data. This is not currently being used but was tested as functional with the (now disabled) Model tests.